package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "test",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
```

Comment:
React 19 was released a few weeks ago, let's upgrade to use that version.

Reply:
I don't think this needed to be a hard blocker on the PR, but I agree we should use the newest version of React.
Looks like the `npm create vite@latest` command hasn't been updated to use React 19 yet. I'll make the change.